en/ios/user-interface.mdown

```diff
@@ -484,8 +484,56 @@ The easiest way to understand this class is with an example. This subclass of `P
 @end
 ```
 ```swift
-// no Swift example
-// Want to contribute to this doc? https://github.com/ParsePlatform/Docs
+class SimpleTableViewController : PFQueryTableViewController {
+    
+    override init(style: UITableViewStyle, className: String?) {
+        super.init(style: style, className: className)
+        parseClassName = "Todo"
+        pullToRefreshEnabled = true
+        paginationEnabled = true
+        objectsPerPage = 25
+    }
+
+    required init?(coder aDecoder: NSCoder!) {
+        super.init(coder: aDecoder)
+        parseClassName = "Todo"
+        pullToRefreshEnabled = true
+        paginationEnabled = true
+        objectsPerPage = 25
+    }
+    
+    override func queryForTable() -> PFQuery {
+        let query = PFQuery(className: self.parseClassName!)
+        
+        // If no objects are loaded in memory, we look to the cache first to fill the table
+        // and then subsequently do a query against the network.
+        if self.objects!.count == 0 {
+            query.cachePolicy = .CacheThenNetwork
+        }
+        
+        query.orderByDescending("createdAt")
+        
+        return query
+    }
+    
+    override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath, object: PFObject?) -> PFTableViewCell? {
+        let cellIdentifier = "cell"
+        
+        var cell = tableView.dequeueReusableCellWithIdentifier(cellIdentifier) as? PFTableViewCell
+        if cell == nil {
```

Comment:
I just came up with a crazy way to do this. Let's not do that, just wanted to put it here because it's crazy...
```swift
let cell = tableView.dequeueReusableCellWithIdentifier(cellIdentifier) as? PFTableViewCell ?? {
            PFTableViewCell(style: .Subtitle, reuseIdentifier: cellIdentifier)
        }()
```